core/src/services/compfs/writer.rs

```diff
@@ -73,6 +85,9 @@ impl oio::Write for CompfsWriter {
     }
 
     async fn abort(&mut self) -> Result<()> {
-        Ok(())
+        Err(Error::new(
+            ErrorKind::Unsupported,
+            "cannot abort completion-based operations",
```

Comment:
The `abort` here does not have the same semantics as in io-uring. Leaving it as `Ok(())` is sufficient.

Reply:
> Aborted writer should not write actual file.

Should I delete the file if writer is aborted...?